_meeting-notes/15-10-19.md

```diff
@@ -0,0 +1,115 @@
+      - Best way to communicate? Place to accumulate docs? (testground repo feels overloaded to me)
+    - Rationales / decision issues
+      - Not sure best place to have these conversations? GitHub might be the wrong place ... not intended to question spec or decisions that have been made
+      - [vs. Testlab](https://github.com/ipfs/testground/issues/64)
+      - [vs. ipfs/benchmarks](https://github.com/ipfs/testground/issues/66)
+      - [Elasticsearch Options](https://github.com/ipfs/testground/issues/67)
+      - [Grafana vs. Kibana](https://github.com/ipfs/testground/issues/68)
+  - Made "playground" repo for experimentation
+  - Booked tickets to Tenerife
+  
+
+## :question: Questions (Blockers, Technical Design needs)
+
+- (@daviddias) How should I go about tests that require ingesting multiple TBs of data? (non-blocker for this week)
+    - Options:
+      - Use EBS snapshot. Mount for the test
+      - Questions:
+        - Will we have infrastructure provider dependent tests?
+- (@daviddias) (might) Need a tip on getting smlbench to work https://github.com/ipfs/testground/pull/60/files#r331433948
+- (@jimpick) Improvements to communication / documentation - I'm trying to restrict myself to GitHub, but unsure of how to structure things...
+  - How to deal with Secrets?
+    - Use <INSERT TOKEN HERE> notes and then let people know (FAQ) that they need 1password
+  - All in one repo for now (docs, examples, tutorials, code)
+
+## :dart: Next
+
+### Top 3 things to get done this week
+
+- Ship Test Plan 1
+- Ship Test Plan 2
+- Get results of both Test Plans outputting to Elastic Search
+
+### Assignments
+
+- ==@daviddias==
+  - [ ] Review & Merge https://github.com/ipfs/testground/pull/33
+  - [ ] Clarify how we use labels and what are the expectations
+  - [ ] Add the Unixfs.Add to https://github.com/ipfs/go-ipfs/pull/6695 and complete the Tutorial section
+  - [ ] Get smlbench plan working.
+  - [ ] Write the tests for Test Plan 1
+    - [ ] `Test:` IPFS Add Defaults
+    - [ ] `Test:` IPFS Add Trickle DAG
+    - [ ] `Test:` IPFS Add Dir Sharding
+    - [ ] `Test:` IPFS MFS Write
+    - [ ] `Test:` IPFS MFS Dir Sharding
+    - [ ] `Test:` IPFS Url Store (not available through the Core API, will need the go-ipfs through IPTB code working first)
+    - [ ] `Test:` IPFS File Store (not available through the Core API, will need the go-ipfs through IPTB code working first)
+    - [ ] Sync with Molly on Testing OKRs on the IPFS Testing OKRs
+    - [ ] Upgrade meeting format to get a reality check with OKRs
+- ==@jimpick==
+  - [ ] Add labels to the questions/issues opened so that Raul (Tech Lead) can better prioritize its time
+  - [ ] Review https://github.com/ipfs/testground/pull/60/files#r331433948 to see if there is any wisdom that can be provided
+  - [ ] Write the tests for Test Plan 2
+    - [ ] `Test:` ??
```

Comment:
@jimpick mind describing the Test Cases here so that I can upgrade trello accordingly

Reply:
For sure. I was editing in Trello live when you deleted everything... :-)

Reply:
It's all in this PR